.eslintrc.js

```diff
@@ -1,45 +1,18 @@
 module.exports = {
-  env: {
-    browser: true,
-    es6: true,
-  },
   extends: [
-    'standard',
-    'standard-react',
-    'prettier/react',
+    'react-app',
+    'prettier/@typescript-eslint',
     'plugin:prettier/recommended',
   ],
-  parser: 'babel-eslint',
-  parserOptions: {
-    ecmaFeatures: {
-      experimentalObjectRestSpread: true,
-      jsx: true,
-    },
-    sourceType: 'module',
-  },
-  plugins: ['prettier', 'react', 'react-hooks'],
+  plugins: ['react-hooks'],
   rules: {
-    'react/prop-types': 'warn',
     'react-hooks/rules-of-hooks': 'error',
     'react-hooks/exhaustive-deps': 'error',
-    'import/no-unresolved': [
-      'error',
-      { ignore: ['^react(-dom)?$', '^styled-components$'] },
-    ],
-    'promise/no-nesting': ['off'],
-    'linebreak-style': ['error', 'unix'],
-    curly: 'error',
   },
   settings: {
     react: {
       pragma: 'React',
-      version: '16.6',
+      version: 'detect',
```

Comment:
👀 woah, this is a thing? It'll resolve your node modules (or look at your package.json)?

Reply:
It is! but following https://github.com/aragon/use-viewport/pull/9#discussion_r496610693 I didn't need to specify this as the cli tool [manages it](https://github.com/formium/tsdx/blob/9cfa396f7df95dc44d46a096ebaeb50289af85f2/src/createEslintConfig.ts#L19-L31).